packages/legend-studio/src/stores/sidebar-state/WorkspaceReviewState.ts

```diff
@@ -262,20 +262,47 @@ export class WorkspaceReviewState {
   ): GeneratorFn<void> {
     this.isCreatingWorkspaceReview = true;
     try {
-      const description =
-        reviewDescription ??
-        `review from ${this.editorStore.applicationStore.config.appName} for workspace ${this.sdlcState.activeWorkspace.workspaceId}`;
-      this.workspaceReview = Review.serialization.fromJson(
-        (yield this.editorStore.sdlcServerClient.createReview(
-          this.sdlcState.activeProject.projectId,
-          {
-            workspaceId: this.sdlcState.activeWorkspace.workspaceId,
-            title,
-            workspaceType: this.sdlcState.activeWorkspace.workspaceType,
-            description,
-          },
-        )) as PlainObject<Review>,
-      );
+      const originalProjectDependencies =
+        this.editorStore.projectConfigurationEditorState
+          .originalProjectConfiguration?.projectDependencies;
+      const listLength = originalProjectDependencies?.length ?? 0;
+      let versionError = false;
+      let i = 0;
+
+      // eslint-disable-next-line
+      const VERSION_REGEX = new RegExp('((0|([1-9]d*)).){2}(0|[1-9]d*)');
```

Comment:
Could we just do a forEach and check the regex of each dependency to have x.x.x and if they don't throw the error. 
This is already covered in a `try` ,`catch` with the error being notified to the user regardless.

Reply:
Also please don't disable `eslint`

Reply:
Like Mauricio said, this check here could be simplified quite a bit.

Reply:
ok.. i'll make the required updates